packages/langium/src/parser/value-converter.ts

```diff
@@ -45,10 +45,14 @@ export class DefaultValueConverter implements ValueConverter {
             case 'STRING': return convertString(input);
             case 'ID': return convertID(input);
             case 'REGEXLITERAL': return convertString(input);
+            case 'BIGINT': return convertBigint(input);
+            case 'DATE': return convertDate(input);
```

Comment:
Yeah they're not critical, I can take them out. If they're desired for some reason, it would be easy to add them in later.

I figured they might be nice for shorter terminals, like `terminal BIGINT: INT` instead of writing out `terminal BIGINT returns bigint: INT`; based on what we have already for `INT`. Since the same can be expressed with the return type, they don't add any expressiveness, just a shorthand equivalent.